docs/pages/faq.mdx

```diff
@@ -114,10 +114,30 @@ Please refer to our [Networking](./reference/networking.mdx) guide.
 
 Teleport offers this feature for the [Enterprise versions of Teleport](choose-an-edition/teleport-enterprise/introduction.mdx).
 
+## Why do I see an alert that some agents are out of date?
+
+Teleport monitors the inventory of all cluster components and compares their
+Teleport versions with the latest release on our GitHub page. If a component is
+not on the latest release, Teleport will create a cluster alert enclouraging
+users to upgrade.
+
+This check is performed against all cluster components, including the Proxy
+Server and Auth Server, as well as agents running other Teleport Services.
+
+## Can I suppress warnings about available upgrades?
+
+Yes. The `tctl alerts ack` command can be used to acklowledge an alert and
+prevent it from being displayed to users. To acknowledge an alert, you need its
```

Comment:
Should we add "temporarily" before "prevent" if they can't be perma-ack'd?

Reply:
Sure. There is no permanent ack, but nothing stops you from using an absurd TTL to achieve the same result.